terraform/table_terraform_resource.go

```diff
@@ -323,22 +384,18 @@ func buildResource(ctx context.Context, isTFFilePath bool, content []byte, path
 			tfResource.DependsOn = s
 
 		case "instances":
-			if reflect.TypeOf(v).String() != "[]interface {}" {
-				return tfResource, fmt.Errorf("The 'instances' argument for resource '%s' must be of type list", name)
-			}
-			for _, v := range v.([]interface{}) {
-				convertedValue := convertModelDocumentToMapInterface(v)
-				cleanedValue := removeKicsLabels(convertedValue).(map[string]interface{})
-				for property, value := range cleanedValue {
-					tfResource.Instances[property] = value
-				}
-			}
 
 		// It's safe to add any remaining arguments since we've already removed all "_kics" arguments
 		default:
 			tfResource.Arguments[k] = v
 		}
 	}
+
+	// Set the arguments as null, if no value (for terraform state files)
+	if len(tfResource.Arguments) == 0 {
```

Comment:
Can you please make similar updates to any `arguments` column and the `lifecycle` column here to be `null` instead of `{}`?